client/integration-tests/custom-templates/src/main/resources/templates/api.qute

```diff
@@ -28,6 +28,12 @@ import io.quarkiverse.openapi.generator.annotations.GeneratedClass;
 import io.quarkiverse.openapi.generator.annotations.GeneratedMethod;
 import io.quarkiverse.openapi.generator.annotations.GeneratedParam;
 
+{#if useBeanValidation}
```

Comment:
Why are you changing this file? It's not related to the `bean-validation` test. It's the custom template used in the `custom-templates` test.